workspaces/spectacle/src/index.ts

```diff
@@ -307,6 +334,10 @@ export async function makeSpectacle(opticContext: IOpticContext) {
       shapeChoices: async (parent: any, args: any, context: any, info: any) => {
         return context.spectacleContext().shapeViewerProjection[args.shapeId];
       },
+      endpoint: async (parent: any, args: any, context: any, info: any) => {
+        const { pathId, method } = args;
+        return context.spectacleContext().getEndpointProjection(pathId, method);
+      },
```

Comment:
Since this is the only field in this schema so far, it's fine to generate the whole thing at once. Eventually, we might want to move each `command` field to its own resolver so the various commands are only generated if the field is requested by a client. In this case the commands are pretty cheap to generate, but that's definitely not always the case.